dev_utils/compute_psf_library.py

```diff
@@ -6,13 +6,19 @@
 import time
 from os.path import abspath, sep, join, exists, isdir, split
 import os
-if not os.environ.get('PYSYN_CDBS'):
-    os.environ['PYSYN_CDBS'] = '/grp/hst/cdbs'
-assert exists(os.environ['PYSYN_CDBS']), "Can't load synthetic photometry files!"
-import pysynphot
 from itertools import product, chain
 import matplotlib
 matplotlib.use('Agg')
+from packaging import version as package_version
+import poppy
+if not os.environ.get('PYSYN_CDBS'):
+    os.environ['PYSYN_CDBS'] = '/grp/hst/cdbs'
+assert exists(os.environ['PYSYN_CDBS']), "Can't load synthetic photometry files!"
+poppy_ver = poppy.__version__
+if package_version.parse(poppy_ver) > package_version.parse("0.9.2"):
+    import stsynphot
+else:
+    import pysynphot
```

Comment:
I wouldn't do this: This looks like a maintenance nightmare. Much easier to cut the next major release and not just pysynphot at all.

Reply:
Agreed in the long run, yes. This is intentional and needed in the short term transitional period however. We plan to take it out after this next release.